x/staking/keeper/keeper.go

```diff
@@ -19,17 +21,18 @@ var _ types.DelegationSet = Keeper{}
 
 // keeper of the staking store
 type Keeper struct {
-	storeKey   sdk.StoreKey
-	cdc        codec.BinaryCodec
-	authKeeper types.AccountKeeper
-	bankKeeper types.BankKeeper
-	hooks      types.StakingHooks
-	paramstore paramtypes.Subspace
+	storeKey     sdk.StoreKey
+	transientKey sdk.StoreKey
```

Comment:
there is a plan to deprecate this store type, could we remove the reliance on it here

Reply:
I've opened this PR that should fix it https://github.com/cosmos/cosmos-sdk/pull/12845

Reply:
seems like are still references to the transientkey in the module and the app.go. Can we remove those as well